tensorflow_serving/model_servers/http_server.cc

```diff
@@ -160,9 +160,20 @@ class RestApiRequestDispatcher {
     string output;
     VLOG(1) << "Processing HTTP request: " << req->http_method() << " "
             << req->uri_path() << " body: " << body.size() << " bytes.";
-    const auto status =
+    auto status =
         handler_->ProcessRequest(req->http_method(), req->uri_path(), body,
                                  &headers, &model_name, &method, &output);
+
+    if (req->http_method() == "OPTIONS") {
+      absl::string_view origin_header = req->GetRequestHeader("Origin");
+      if (RE2::PartialMatch(origin_header, "https?://")) {
```

Comment:
start with https://

Reply:
I was hoping that local testing apps (eg. origin = http://localhost:4200) can directly call this API.
Are we going to enforce https ?

Reply:
Okay.